source/backend/api/Services/FormDocumentService.cs

```diff
@@ -60,6 +60,47 @@ public IList<PimsFormType> GetFormDocumentTypes(string formTypeCode)
             return new List<PimsFormType>() { _formTypeRepository.GetByFormTypeCode(formTypeCode) };
         }
 
+        public async Task UploadFormDocumentTemplate(string formTypeCode, DocumentUploadRequest uploadRequest)
+        {
+            throw new NotImplementedException();
```

Comment:
Can you explain what is going on here? generally we do no leave blocks of commented-out code in the application.

Reply:
removed, this was part of draft.